src/themes/saas/components/checkbox.ts

```diff
@@ -1,26 +1,93 @@
 import { checkboxAnatomy } from '@chakra-ui/anatomy'
-import { createMultiStyleConfigHelpers } from '@chakra-ui/react'
+import { createMultiStyleConfigHelpers, defineStyle } from '@chakra-ui/react'
+import { mode } from '@chakra-ui/theme-tools'
 
-const { defineMultiStyleConfig } = createMultiStyleConfigHelpers(checkboxAnatomy.keys)
+const { definePartsStyle, defineMultiStyleConfig } = createMultiStyleConfigHelpers(checkboxAnatomy.keys)
 
+// Manteniendo la estructura original
```

Comment:
Do not add comments in languages other than English, but also... this comment looks GPT-suspicious